authorization-service/serverless.ts

```diff
@@ -0,0 +1,46 @@
+import { AWS } from '@serverless/typescript';
+import { basicAuthorizer } from '@functions/basicAuthorizer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+console.log(`Debug level logs enabled: ${process.env.ENV_DEBUG || 'false'}`);
+
+const serverlessConfiguration: AWS = {
+  service: 'authorization2',
+  frameworkVersion: '2',
+  useDotenv: true,
+  custom: {
+    serverlessOffline: {
+      httpPort: 2007
+    },
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true,
+    },
+  },
+  plugins: [
+    'serverless-webpack',
+    'serverless-offline'
+  ],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    stage: '${opts:stage, \'dev\'}',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      ENV_DEBUG: process.env.ENV_DEBUG || 'false',  // I got strange issue: ${opts:} didn't work here!
+      akondratsky: 'TEST_PASSWORD',
```

Comment:
we shouldn't save login and password there, I guess better use env file for that

Reply:
I believe we shouldn't even in env file. I just keep in mind that this course not about security :)
Sorry for the late reply, I missed your comment